lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -71,16 +46,9 @@ def define
         option :tax_credits
         option :income_support
 
-        save_input_as :benefit
-
-        calculate :how_long_question_titles do
-          if benefit == "disability_benefits"
-            "How long will you be abroad for?"
-          elsif going_or_already_abroad == "going_abroad"
-            "How long are you going abroad for?"
-          else
-            "How long will you be living abroad for?"
-          end
+        on_response do |response|
+          self.benefit = response
+          calculator.benefit = benefit
```

Comment:
```suggestion
          calculator.benefit = response
```

Again for consistency with other smart answers - I think we can just save the state to the calculator.

Reply:
Thank you - I've moved it to the calculator